[SRVOCF-440] Reorg, add, & update content on page "Getting started with functions" (#51029)
openshift/openshift-docs: modules/serverless-kn-func-delete.adoc

// Module included in the following assemblies

// * serverless/cli_tools/kn-func-ref.adoc

:_content-type: PROCEDURE
[id="serverless-kn-func-delete_{context}"]
= Deleting a function

> You can delete a function from your cluster by using the `kn func delete` command. This is useful when a function is no longer a part of your solution. By deleting a function, you can save resources and keep your cluster and solution cleaner.

Reviewer: Not sure what the value is of the additional text here really, sort of feels like fluff.

Author: Well since the docs guidelines call for explicit Why, here's the Why. The Why for deleting something is obvious, and hence feels like fluff. -_o_-

Author: [Used your suggestion]

> You can delete a function from your cluster by using the `kn func delete` command. This is useful when a function is no longer a part of your solution. By deleting a function, you can save resources and keep your cluster and solution cleaner.

Reviewer: ```suggestion
You can delete a function by using the `kn func delete` command. This is useful when a function is no longer required, and can help to save resources on your cluster.
```
